app/pulp/app/viewsets/repository.py

```diff
@@ -124,6 +124,16 @@ class PublisherViewSet(NamedModelViewSet):
     queryset = Publisher.objects.all()
     filter_class = PublisherFilter
 
+    def destroy(self, request, pk):
```

Comment:
i would suggest to put this note probably somewhere here https://github.com/pulp/pulp/blob/3.0-dev/docs/contributing/3.0-development/rest-api.rst#lookup_field-annoyances